app/controllers/application_controller/ci_processing.rb

```diff
@@ -857,8 +857,27 @@ def show_details(db, options = {})  # Pass in the db, parent vm is in @vm
       # Came in from outside, use RJS to redraw gtl partial
       if params[:ppsetting] || params[:entry] || params[:sort_choice]
         replace_gtl_main_div
+      elsif request.xml_http_request?
+        # reload toolbars - AJAX request
+        c_tb = build_toolbar(center_toolbar_filename)
+        render :update do |page|
+          page << javascript_prologue
+          page.replace("flash_msg_div", :partial => "layouts/flash_msg")
+          page.replace_html("main_div", :partial => "shared/views/ems_common/show") # Replace main div area contents
+          page << javascript_pf_toolbar_reload('center_tb', c_tb)
+          page.replace_html("paging_div",
+                            :partial => 'layouts/pagingcontrols',
+                            :locals  => {:pages      => @pages,
+                                         :action_url => @lastaction,
+                                         :db         => @view.db,
+                                         :headers    => @view.headers})
+        end
       else
-        render :action => 'show'
+        if controller_name == "ems_cloud"
+          render :template => "shared/views/ems_common/show"
```

Comment:
@himdel This method is only called when clicking on a link to list of associations(i.e link to list of Accounts, patches etc from Host/VM summary, or Arbitration Profiles in this case from cloud provider summary) from a summary screen, since show partials was removed from views/ems_cloud that's the only one getting affected in this case. Host/VM screens should be fine.